test/topology_custom/test_replace_with_encription.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import pytest
+from test.pylib.manager_client import ManagerClient
+from test.pylib.scylla_cluster import ReplaceConfig
+
+@pytest.mark.asyncio
+async def test_replace_with_encription(manager: ManagerClient):
+    s1 = await manager.servers_add(3,
```

Comment:
It would be nice to have some doc string with clarification what we are testing.